packages/babel-config/src/api.ts

```diff
@@ -36,6 +36,7 @@ export const getApiSideBabelPresets = (
       },
       'rwjs-babel-preset-typescript',
     ],
+    // TODO(jgmw): Search for presetEnv = true usage in all this babel preset stuff
```

Comment:
✅

Reply:
Maybe we can remove the presetEnv option, and rename this function to make it clear that we only use `getApiSideBabelPresets` for "runtime transformation" e.g. jest, exec, possibly data-migrations